test/index.test.js

```diff
@@ -14,26 +14,46 @@ const rimraf = promisify(_rimraf);
 const FIXTURES_DIR = `${__dirname}/fixtures`;
 const DEFAULT_SCRIPT = 'microbundle';
 
-const times = (n, fn) => Array.from({ length: n }).map(i => fn(i));
 const join = (arr, delimiter = '') => arr.join(delimiter);
-const constant = konst => () => konst;
 
 const printTree = (nodes, indentLevel = 0) => {
-	const indent = join(times(indentLevel, constant('  ')));
+	const indent = '  '.repeat(indentLevel);
 	return join(
 		nodes
 			.filter(node => node.name[0] !== '.')
-			.map(
-				node =>
-					`${indent}${node.name}\n${
-						node.type === 'directory'
-							? printTree(node.children, indentLevel + 1)
-							: ''
-					}`,
-			),
+			.map(node => {
+				const isDir = node.type === 'directory';
+				return `${indent}${node.name}\n${
+					isDir ? printTree(node.children, indentLevel + 1) : ''
+				}`;
+			}),
 	);
 };
 
+const getBuildScript = async (fixturePath, defaultScript) => {
+	const pkgJsonPath = resolve(fixturePath, 'package.json');
+
+	try {
+		await stat(pkgJsonPath);
```

Comment:
according to - https://nodejs.org/api/fs.html#fs_fs_stat_path_options_callback 
> Using fs.stat() to check for the existence of a file before calling fs.open(), fs.readFile() or fs.writeFile() is not recommended. Instead, user code should open/read/write the file directly and handle the error raised if the file is not available.

Reply:
Yes, I had also read it. What do we do?

Reply:
```js
let pkg = {};
try {
  pkg = JSON.parse(await readFile(pkgJsonPath, 'utf8'));
} catch (e) {}
return pkg && pkg.scripts && pkg.scripts.build;